packages/vscode/package.json

```
@@ -145,6 +145,12 @@
             "/path/to/your/schema.prisma"
           ],
           "description": "If you have a Prisma schema file in a custom path, you will need to provide said path `/path/to/your/schema.prisma` to run generate"
+        },
+        "prisma.pdpNotification": {
+          "title": "Pdp Notification",
```

```suggestion
          "title": "Display promotion for Prisma Data Platform on next extension activation",
```
How about this @Jolg42 @Druue?

---

Sounds good!